v3/src/components/free-tile-row.tsx

```diff
@@ -14,24 +14,87 @@ interface IFreeTileRowProps {
   getTile: (tileId: string) => ITileModel | undefined
 }
 export const FreeTileRowComponent = observer(({ content, row, getTile }: IFreeTileRowProps) => {
+  const [resizingTileStyle, setResizingTileStyle] =
+    useState<{left: number, top: number, width: number, height: number}>()
+  const [resizingTileId, setResizingTileId] = useState("")
+
   const handleCloseTile = (tileId: string) => {
     if (!tileId) return
     content?.deleteTile(tileId)
   }
+
+  const handleResizePointerDown = (e: React.PointerEvent, tile: IFreeTileLayout, direction: string) => {
+    const startWidth = tile.width
+    const startHeight = tile.height
+    const startPosition = {x: e.pageX, y: e.pageY}
+
+    let resizingWidth = startWidth, resizingHeight = startHeight, resizingLeft = tile.x
+    // Because user can start drag 8px within the border, the component's startPosition.x moves by number of pixels
+    // the pointer down event location, which moves the entire component to the right by the same number of pixels.
+    // So we force it to always be the left position of the component
+    const startLeft = startPosition.x > tile.x ? tile.x : startPosition.x
```

Comment:
Given the use of deltas below, I think this should just be `startLeft = tile.x` at this point. Did that not work as expected?